dags/utils/gcp.py

```diff
@@ -385,6 +387,9 @@ def bigquery_etl_query(
     destination_table,
     parameters=(),
     arguments=(),
+    dataset_id="telemetry",
```

Comment:
This is not marked as Optional below, but there's a default here. I think it makes sense to remove the default and make this a required param.

Reply:
Or maybe the default here can be a transitional thing until we have all derived tables moved into the telemetry_derived locations.

Reply:
let's remove it now, just in case we don't get another chance